### DISCUSSION TITLE
Question about GH Student Developer Pack

### DISCUSSION THREAD

**Discussion by AfterNetwork** [community/community#21805] [New to GitHub]
<p>Can I use an unofficial transcript to verify academic status?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/afternetwork">@afternetwork</a>,</p>
<p>Thanks for being here! As, far as I know they do as long as the current month/year in which you are applying are reflected. Maybe <a class="mention" href="/sanicki">@sanicki</a> can chime in with some more info. </p>

**Comment by sanicki** [comment]
<p>Hi <a class="mention" href="/afternetwork">@afternetwork</a>,</p>
<p><a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> is correct: Whatever documentation you provide should be dated to demonstrate your current student status. So long as they do, your unofficial transcripts should be sufficient. If not, you will have the opportunity to apply again with additional proof. We consider all previously-submitted materials when evaluating your current application.</p>
<p>Cheers,</p>
<p><a class="mention" href="/sanicki">@sanicki</a> </p>

**Comment by hjrsmail** [comment]
If I have graduated from college but my campus account is still active, can I use the student pack?
